sds_data_manager/lambda_code/SDSCode/dependency_config_ultra.py

```diff
@@ -16,80 +16,142 @@
     PreProcessingDependency(
         primary_instrument="ultra",
         primary_data_level="l0",
-        primary_descriptor="raw",
+        primary_descriptor="45raw",
```

Comment:
```suggestion
        primary_descriptor="raw",
```

Reply:
Wouldn't this make it to where if only packets from Ultra 45 arrive then it would kick off Ultra 45 and Ultra 90 l1a processing?